tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -805,6 +980,35 @@ class MklConvOp : public OpKernel {
     return nullptr;
   }
 
+#ifdef ENABLE_MKLDNN_V1
+  virtual void AllocateOutputTensor(OpKernelContext* context,
+                                    const ConvFwdPd& conv_prim_desc,
+                                    const memory::dims& output_dims_mkl_order,
+                                    MklTensorFormat output_tf_format,
```

Comment:
Can define `MklTensorFormat` vs `memory::format` and eliminate a bit more redundancy.

Reply:
Done.